code/modules/mob/living/brain/brain_item.dm

```diff
@@ -18,12 +18,12 @@
 	vital = FALSE
 	decoy_override = TRUE
 
-/obj/item/organ/brain/Insert(mob/living/carbon/C, special = 0)
+/obj/item/organ/brain/Insert(mob/living/carbon/C, special = 0, no_id_transfer = FALSE)
```

Comment:
Was it you or me that added `no_id_transfer`? It doesn't show up anywhere else in code and I don't think it's ever called properly...

Reply:
Ah, it's in the species change code. All good.